velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -414,21 +416,53 @@ class NonNumericMinMaxAggregateBase : public exec::Aggregate {
 
     auto accumulator = value<SingleValueAccumulator>(group);
     rows.applyToSelected([&](vector_size_t i) {
-      if (decoded.isNullAt(i)) {
+      if (checkNulls(decoded, i)) {
         return;
       }
+
       if (!accumulator->hasValue() ||
           compareTest(compare(accumulator, decoded, i))) {
         accumulator->write(baseVector, indices[i], allocator_);
       }
     });
   }
+
+  FOLLY_ALWAYS_INLINE static void checkNestedNull(
```

Comment:
This function appears unused. Let's remove.

Reply:
Removed